content/articles/a-beginners-guide-on-how-to-become-a-blockchain-developer/index.md

```diff
@@ -0,0 +1,181 @@
+
+This is a collection of libraries that allows you to interact with the Ethereum node either remotely or locally using HTTP, Websocket, or IPC.
+
+2. [React.js](https://reactjs.org/)
+
+This is an open-source javascript library that enables programmers to build beautiful user interfaces. You can use React.js to build those beautiful user interfaces on the front-end.
+
+3. [Ethers.js](https://docs.ethers.io/v5/)
+
+It is a collection of libraries that allows you to interact with the Ethereum blockchain and its ecosystem.
+
+4. [Metamask](https://metamask.io/)
+
+It is a browser extension that allows a user to access their Ethereum wallets on the Ethereum blockchain. Users can buy, store, and swap ether tokens using this library.
+
+5. [Node.js](https://nodejs.org/en/)
+
+It is a back-end JavaScript runtime environment that executes javascript code outside a web browser. Decentralized applications usually use node.js to interact with smart contracts and public nodes.
+
+6. [Ganache](https://www.trufflesuite.com/ganache)
+
+It is a library developers can use for Ethereum blockchain development. It enables developers to develop, deploy, and test decentralized applications in a safe and deterministic environment without interacting with the main Ethereum blockchain.
+
+### Which programming languages for blockchain?
+
+1. Solidity
+
+This is the programming language that is used for creating Ethereum smart contracts.
+
+2. Javascript
+
+It is used primarily for creating user interfaces. 
+
+3. Go and C++
+
+These two languages are used by the blockchain core developers to building the back-ends of blockchain systems.
+
+4. Haskell
+
+This is the programming language used by core blockchain developers in building the Cardano blockchain.
+
+#### Note
+
+Solidity is still currently the most popular programming language of choice for many developers. This is due to its large developer community. Unless you have a specific use case or you're a blockchain core developer, please stick to Solidity.
+
+### Use cases for blockchain
+
+1. Finance
+
+#### Cryptocurrencies
+
+Today, there exist over 5,000 cryptocurrencies. This is according to [CoinMarketCap](https://coinmarketcap.com/), a company that tracks cryptocurrencies' market caps, rankings, prices, exchange volumes, and currency conversion. However, the most popular cryptocurrencies in this space include [Bitcoin](https://bitcoin.org/en/), [Ethereum](https://ethereum.org/en/), [Cardano](https://cardano.org/) and [Polkadot](https://polkadot.network/).
+
+People tend to think that cryptocurrencies are the only blockchain's use case. This is not the case. But, it is the biggest use case for blockchain. 
+
+As a blockchain developer, you can build and launch one of your cryptocurrencies. But there is a catch. With the high number of cryptocurrencies out there, it is advisable to build and launch a coin that tries to solve real-world problems. Otherwise, it'll just exist and not be adopted by anyone. 
```

Comment:
```suggestion
As a blockchain developer, you can build and launch a token. You may even go all the way and build your own cryptocurrency. But there is a catch. With the high number of cryptocurrencies and tokens out there, it is advisable to develop a token (or cryptocurrency if feasible) that tries to solve real-world problems. Otherwise, it'll be hard to gain traction or survive in the long term.
```

Reply:
I've made a suggestion on this line. Perhaps add a link that distinguishes between a token and a coin. And a single line mentioning the challenges of creating a cryptocurrency such as the resources needed and distrust from potential users due to the reputation of ICOs.